src/html5sortable.ts

```diff
@@ -364,8 +380,27 @@ export default function sortable (sortableElements, options: configuration|objec
             originalTarget: target
           }
         }))
-      }
 
+        _on(sortableContainer, 'dragleave', e => {
+          const outTarget = e.releatedTarget || e.fromElement
```

Comment:
This definitely has to be fixed to relatedTarget (there is a typo). e.fromElement is for very old browsers, but maybe we just add a comment // e.fromElement for very old browsers, similar to relatedTarget.

Maybe you can also add a comment above // TODO: rename outTarget to be more self-explanatory-

Basically I think it is the parent element (not sortable) of the sortable that the item was in. We are checking that it actually is moved to a different sortable and does not stay within.